offchain-modules/packages/scripts/src/integration.ts

```diff
@@ -177,6 +203,13 @@ async function startService(
     )}`,
     false,
   );
+  await execShellCmd(
```

Comment:
What's the point of add monitor to integration test?
IMO, it's a background process and have no influence to the test.

Reply:
OK，I will remove it from integration test